src/components/Button/Button.test.tsx

```diff
@@ -1,7 +1,8 @@
 /*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
 import * as React from 'react';
 import { shallow, ShallowWrapper } from 'enzyme';
-import Button, { ButtonProps, ButtonColor, ButtonStatus, ButtonIconPosition, ButtonSize } from '.';
+import Button,
+  { ButtonProps, ButtonColor, ButtonStatus, ButtonIconPosition, ButtonSize, ButtonType } from '.';
```

Comment:
Do you use Prettier for code formatting?

Reply:
I've upped the line limit to 120 to be consistent with yamjs. :-)

Reply:
(and now have prettier)